BGL/include/CGAL/boost/graph/helpers.h

```diff
@@ -777,6 +778,22 @@ void clear(FaceGraph& g)
   CGAL_postcondition(num_faces(g) == 0);
 }
 
+/**
+* \ingroup PkgBGLHelperFct
+*
+* checks whether the graph is empty, by checking that it does not contain any vertex.
+*
+* @tparam FaceGraph model of `FaceGraph`
+*
+* @param g the graph to test
+*
+**/
+template<typename FaceGraph>
+bool is_empty(const FaceGraph& g)
+{
+  return boost::empty(vertices(g));
+}
```

Comment:
This function has created a new build error in Mesh_3:
https://cgal.geometryfactory.com/CGAL/testsuite/CGAL-4.11-Ic-65/Mesh_3_Examples/TestReport_lrineau_CentOS6-CXX11-Boost157.gz
```
In file included from /home/cgal_tester/build/src/cmake/platforms/CentOS6-CXX11-Boost157/test/Mesh_3_Examples/mesh_polyhedral_domain_with_surface_inside.cpp:8:
/usr/lib/gcc/x86_64-redhat-linux/4.4.7/../../../../include/c++/4.4.7/tr1_impl/type_traits: In constructor ‘CGAL::Polyhedral_mesh_domain_with_features_3<IGT_, Polyhedron_, TriangleAccessor, Patch_id, Use_exact_intersection_construction_tag>::Polyhedral_mesh_domain_with_features_3(const Polyhedron_&, const Polyhedron_&, CGAL::Random*) [with IGT_ = CGAL::Epick, Polyhedron_ = CGAL::Polyhedron_3<CGAL::Epick, CGAL::Mesh_3::Mesh_polyhedron_items<int>, CGAL::HalfedgeDS_default, std::allocator<int> >, TriangleAccessor = CGAL::Triangle_accessor_3<CGAL::Polyhedron_3<CGAL::Epick, CGAL::Mesh_3::Mesh_polyhedron_items<int>, CGAL::HalfedgeDS_default, std::allocator<int> >, CGAL::Epick>, Patch_id = int, Use_exact_intersection_construction_tag = CGAL::Boolean_tag<true>]’:
/home/cgal_tester/build/src/cmake/platforms/CentOS6-CXX11-Boost157/test/Mesh_3_Examples/mesh_polyhedral_domain_with_surface_inside.cpp:50:   instantiated from here
/usr/lib/gcc/x86_64-redhat-linux/4.4.7/../../../../include/c++/4.4.7/tr1_impl/type_traits:326: error: ‘template<class _Tp> struct std::is_empty’ is not a function,
/mnt/testsuite/include/CGAL/boost/graph/helpers.h:1172: error:   conflict with ‘template<class FaceGraph> bool CGAL::is_empty(const FaceGraph&)’
/mnt/testsuite/include/CGAL/Polyhedral_mesh_domain_with_features_3.h:287: error:   in call to ‘is_empty’
```